### DISCUSSION TITLE
Does charliecloud support the Intel GPU?

### DISCUSSION THREAD

**Discussion by b02330224** [hpc/charliecloud#1480] [General]
Does charliecloud support the Intel GPU? If not，do you have plan to support the Intel GPU?
we want to use charliecloud  running AI code on Intel GPU.
Thank you very much!

**Comment by reidpr** [comment]
I'm not familiar with what specifically the Intel GPUs need, but if it's injecting stuff from the host like nVidia, `ch-fromhost` would be able to do that.

We would welcome a PR to add an e.g. `--intel-gpu` switch to `ch-fromhost` to package up the knowledge about what to do for Intel.
